src/w3c/headers.js

```diff
@@ -109,137 +110,96 @@ const W3CDate = new Intl.DateTimeFormat(["en-AU"], {
 });
 
 hb.registerHelper("showPeople", function(name, items = []) {
+  const self = this;
+  const html = (...args) => hyperHTML.wire()(...args);
   // stuff to handle RDFa
-  var re = "",
-    rp = "",
-    rm = "",
-    rn = "",
-    rwu = "",
-    rpu = "",
-    bn = "",
-    editorid = "",
-    propSeeAlso = "";
-  if (this.doRDFa) {
-    if (name === "Editor") {
-      bn = "_:editor0";
-      re = " property='bibo:editor' resource='" + bn + "'";
-      rp = " property='rdf:first' typeof='foaf:Person'";
-    } else if (name === "Author") {
-      rp = " property='dc:contributor' typeof='foaf:Person'";
-    }
-    rn = " property='foaf:name'";
-    rm = " property='foaf:mbox'";
-    rwu = " property='foaf:workplaceHomepage'";
-    rpu = " property='foaf:homepage'";
-    propSeeAlso = " property='rdfs:seeAlso'";
-  }
-  var ret = "";
-  for (var i = 0, n = items.length; i < n; i++) {
-    var p = items[i];
-    if (p.w3cid) {
-      editorid = " data-editor-id='" + parseInt(p.w3cid, 10) + "'";
-    }
-    if (this.doRDFa) {
-      ret +=
-        "<dd class='p-author h-card vcard' " +
-        re +
-        editorid +
-        "><span" +
-        rp +
-        ">";
-      if (name === "Editor") {
-        // Update to next sequence in rdf:List
-        bn = i < n - 1 ? "_:editor" + (i + 1) : "rdf:nil";
-        re = " resource='" + bn + "'";
-      }
-    } else {
-      ret += "<dd class='p-author h-card vcard'" + editorid + ">";
-    }
+  const attr = Object.freeze(self.doRDFa ? {
+    ...name === "Editor" ? {
+      rp1: "rdf:first",
+      rp2: "foaf:Person"
```

Comment:
Based on the discussion in the related bug, it seems safe to remove this stuff.

Reply:
Oh, was that the conclusion? I thought https://github.com/w3c/respec/issues/1503#issuecomment-365561010 is saying that Google actively uses this.

Reply:
Yes and no :) Consider:

> "Whether the specific vocabulary and data patterns you're publishing map to a current public facing question is another matter."

Ideally, we should only include things that *do map* to "rich snippets". None of these seem to show up in any "rich snippets", so IMO they should go.

Reply:
I made a proposal in the bug: when refactoring code, if we find stuff that doesn't show up as a rich snippet, it should be removed from ReSpec (as it has little, to no, value). However, we should keep anything that does show up.